package com.asesoria.contable.app_ac.exceptions;

public class UsuarioNotFoundException extends RuntimeException{
}
